Mobile/CarWashService/Utils.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Utils.swift
+//  CarWashService
+//
+//  Created by Anna Boykova on 17/11/2019.
+//  Copyright © 2019 VXDESIGN.STORE. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+//Constants region
+    let PrimaryColor = hexStringToUIColor(hex:"#631D76")
+    let SuccessColor = hexStringToUIColor(hex:"#60992D")
+    let WarningColor = hexStringToUIColor(hex:"#E56D44")
+    let DangerColor = hexStringToUIColor(hex:"#B22345")
+    let InfoColor = hexStringToUIColor(hex:"#4A58B5")
+    
+    let AppLogoStr = "Car Wash"
```

Comment:
I added "CAR WASH SYSTEM" as app title for web. Possibly it can be unified for web and mobile like that:

- Car Wash: Client
- Car Wash: Company